uv/mlflow/reporter.py

```diff
@@ -18,9 +18,10 @@
 import mlflow as mlf
 from mlflow.entities import Param, Metric, RunTag
 import time
-from typing import Optional, Dict, List
+from typing import Optional, Dict, List, Union
 import uv.reporter.base as b
 import uv.types as t
+from uv.util import attachment
```

Comment:
I prefer not to do this for modules. Can you do `import uv.util.attachment as ua`, then use `ua.flatten` below?